iscript/src/iscript/mac.py

```diff
@@ -478,9 +479,18 @@ def get_app_paths(config, task):
     all_paths = []
     for upstream_artifact_info in task["payload"]["upstreamArtifacts"]:
         for subpath in upstream_artifact_info["paths"]:
-            orig_path = get_artifact_path(upstream_artifact_info["taskId"], subpath, work_dir=config["work_dir"])
             formats = upstream_artifact_info["formats"]
-            app = App(orig_path=orig_path, formats=formats, artifact_prefix=_get_artifact_prefix(subpath))
+            if not formats:
+                # Signing resources don't have a signing format
+                continue
```

Comment:
Isn't it better for this case to raise rather than silently skip?

Reply:
By default, iscript used to sign all items in `upstreamArtifacts`.
With this change, items with an empty `formats` field will be skipped and not signed.
This allows signing resources (entitlements, libconstraints, etc) to be included in `upstreamArtifacts` for downloading from the build task without having iscript sign them.